### PR TITLE
Added subgraph endpoints in facilitator init

### LINKED ISSUE #137: Add origin and auxiliary subgraph endpoints in facilitator init command
Facilitator init command should have origin and auxiliary subgraph ws and rpc endpoints.

### PR DESCRIPTION
**Pull request by gulshanvasnani**
PR contains below changes to add cli options in facilitator command :-
1. `--origin-graph-ws` represents origin ws subgraph endpoint.
2. `--origin-graph-rpc` represents origin rpc subgraph endpoint.
3. `--auxiliary-graph-ws` represents auxiliary ws subgraph endpoint.
4. `--auxiliary-graph-rpc` represents auxiliary ws subgraph endpoint.

Also, updated integration tests of facilitator init.

fixes #137

### CODE REVIEW

File: src/bin/facilitator-init.ts [hunk 1]
```diff
@@ -8,14 +8,18 @@ import Utils from '../Utils';
...
+  .option('-o, --origin-password <origin-password>', 'origin chain account password')
+  .option('-a, --auxiliary-password <auxiliary-password>', 'auxiliary chain account password')
+  .option('-r, --origin-rpc <origin-rpc>', 'origin chain rpc')
+  .option('-h, --auxiliary-rpc <auxiliary-rpc>', 'auxiliary chain rpc')
+  .option('-e, --origin-graph-ws <origin-graph-ws>', 'origin ws subgraph endpoint ')
+  .option('-g, --origin-graph-rpc <origin-graph-rpc>', 'origin rpc subgraph endpoint')
+  .option('-s, --auxiliary-graph-ws <auxiliary-graph-ws>', 'auxiliary ws subgraph endpoint')
+  .option('-i, --auxiliary-graph-rpc <auxiliary-graph-rpc>', 'auxiliary rpc subgraph endpoint')
+  .option('-d, --db-path <db-path>', 'path where db path is present')
+  .option('-p, --force', 'forcefully override facilitator config')
```
Comment (deepesh-kn) [inline src/bin/facilitator-init.ts]: `-f` was correct.

File: src/bin/facilitator-start.ts [hunk 2]
```diff
@@ -20,8 +20,8 @@ process.on('SIGINT', terminationHandler);
 process.on('SIGTERM', terminationHandler);
 
 facilitatorCmd
-  .option('-mc, --mosaic-config <mosaicConfig>', 'path to mosaic configuration')
-  .option('-fc, --facilitator-config <facilitatorConfig>', 'path to facilitator configuration')
+  .option('-m, --mosaic-config <mosaicConfig>', 'path to mosaic configuration')
+  .option('-f, --facilitator-config <facilitatorConfig>', 'path to facilitator configuration')
```
Comment (deepesh-kn) [inline src/bin/facilitator-start.ts]: use `-c`.

Comment (gulshanvasnani) [reply src/bin/facilitator-start.ts]: Using `-t` as `-c` is used in facilitator-init for chain id.

File: src/bin/facilitator-init.ts [hunk 3]
```diff
@@ -92,25 +117,44 @@ commander
...
     facilitatorConfig.database.path = dbPath;
     facilitatorConfig.originChain = originChainId;
-    facilitatorConfig.auxChainId = options.chainId;
-    const setFacilitator = (chainId: string, rpc: string, password: string): void => {
-      const account: Account = Account.create(new Web3(null), password);
-
-      facilitatorConfig.chains[chainId] = new Chain(rpc, account.address);
+    facilitatorConfig.auxChainId = auxChainId;
+    const setFacilitator = (
+      auxChainId: string,
```
Comment (deepesh-kn) [inline src/bin/facilitator-init.ts]: This has to be `chainId` and not auxChainId.
See line 136.

File: README.md [hunk 4]
```diff
@@ -7,7 +7,7 @@ Facilitator init command will create facilitator config for an auxiliary chain.
 A simple run would be the following:
 
 ```
-./facilitator init --mosaic-config <mosaic-config> --chain-id <chain-id> --origin-password <origin-password> --auxiliary-password <auxiliary-password> --origin-rpc <origin-rpc> --auxiliary-rpc <auxiliary-rpc> --db-path <db-path> --force
+./facilitator init --mosaic-config <mosaic-config> --chain-id <chain-id> --origin-password <origin-password> --auxiliary-password <auxiliary-password> --origin-rpc <origin-rpc> --auxiliary-rpc <auxiliary-rpc> --origin-graph-ws <origin-graph-ws> --origin-graph-rpc <origin-graph-rpc> --auxiliary-graph-ws <auxiliary-graph-ws> --auxiliary-graph-rpc <auxiliary-graph-rpc> --db-path <db-path> --force
```
Comment (deepesh-kn) [inline README.md]: In code `--chain-id` is changed to `--aux-chain-id`, Read me still says `--chain-id`.

**Pull request merged**